apps/typegpu-docs/src/components/ControlPanel.tsx

```diff
@@ -30,7 +30,7 @@ function ToggleRow({
 
       <label className="grid items-center justify-end h-10 cursor-pointer">
         <Toggle
-          checked={value}
+          checked={value ?? false}
```

Comment:
Tiny nitpick, we can assign the default value at the very beginning when destructuring props.

```ts
{
  label,
  initial = false,
  onChange,
}
```

```suggestion
          checked={value}
```